.travis.deploy.sh

```diff
@@ -18,5 +18,5 @@ cp .travis.settings.xml ${HOME}/.m2/settings.xml
 mvn releaser:release
 
 # Push latest docker image
-docker login -u="${DOCKER_USER}" -p="${DOCKER_PASSWORD}"
+echo "$DOCKER_PASSWORD" | docker login -u "$DOCKER_USERNAME" --password-stdin
```

Comment:
For safety, do you want to wrap this in a set +x echo set -x . Guessing travis probably hides this anyway, but just a thought.

Reply:
why remove curlies?

Reply:
Secrets are hidden in Travis

Reply:
I'll add curlies for curliness without running`curl` :smile:

Reply:
I'll wrap it cuz why not

Reply:
`curly` `+x-x` FTW